.huskyrc

```diff
@@ -0,0 +1,3 @@
+---
+hooks:
+  pre-commit: lint-staged
```

Comment:
I couldn't find any documentation on .huskyrc. Does this work? The docs show putting configuration in package.json.